secrets-localhost.properties.SAMPLE

```diff
@@ -2,7 +2,7 @@ app.namespace=https://YOUR-HEROKU-APP.herokuapp.com
 app.admin.emails=phtcon@ucsb.edu
 app.member.hosted-domain=ucsb.edu
 
-auth0.domain=YOUR-AUTH0-TENANT.us.auth0.com
+auth0.domain=cs156-s21-6pm-4.us.auth0.com
```

Comment:
As this is a sample file, we need to keep the default values so users can edit them later on for their purposes. Please revert this back to the original default value.